zhaquirks/smartwings/wm25lz.py

```diff
@@ -0,0 +1,105 @@
+"""Device handler for Smartwings blinds."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    Ota,
+    PowerConfiguration,
+    Scenes,
+)
+
+from zhaquirks import DoublingPowerConfigurationCluster
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class InvertedWindowCoveringCluster(CustomCluster, WindowCovering):
+    """WindowCovering cluster implementation.
+
+    This implementation inverts the reported covering percent for non standard
+    devices that don't follow the reporting spec.
+    """
+
+    cluster_id = WindowCovering.cluster_id
+    CMD_GO_UP = 0x0001  # reported as down_close in ZHA commands
+    CMD_GO_DOWN = 0x0000  # reported as up_open in ZHA commands
```

Comment:
Hmm, per ZCL the commands are the other way around (like mentioned in the comments): [WindowCovering cluster zigpy link](https://github.com/zigpy/zigpy/blob/be5b2914a12ac6a19af80c5a83dab40d7957aa01/zigpy/zcl/clusters/closures.py#L684-L685)
Maybe it makes more sense to name them to what they should be here according to ZCL?

(and not really important, but I think you could also use ``WindowCovering.commands_by_name["up_open"].id`` and same for "down_close")

Reply:
Do you mean changing this to:
```
up_open = WindowCovering.commands_by_name["up_down"].id
down_close = WindowCovering.commands_by_name["up_open"].id
```

Reply:
Yeah. It's not really needed, but might be nice not to hard-code cluster attributes/commands further.
I'd still keep them as constants though (as they're each referenced to twice in the code)
```python
CMD_GO_UP = WindowCovering.commands_by_name["up_down"].id
CMD_GO_DOWN = WindowCovering.commands_by_name["up_open"].id
```
There, the constants would be named after ZCL/so what ZHA reports.